content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -174,7 +174,9 @@ as a learner and promoted to a voting member only after the etcd data are fully
 `PublicKeysECDSA`
 : Can be used to create a cluster that uses ECDSA certificates instead of the default RSA algorithm.
 Renewal of existing ECDSA certificates is also supported using `kubeadm certs renew`, but you cannot
-switch between the RSA and ECDSA algorithms on the fly or during upgrades.
+switch between the RSA and ECDSA algorithms on the fly or during upgrades. In versions prior to
+v1.31 this feature has a bug where keys in kubeconfig files will use RSA despite the feature gate
+being enabled.
```

Comment:
You can merge this into _main_ (and also request backports)
```markdown
switch between the RSA and ECDSA algorithms on the fly or during upgrades.
Kubernetes {{< skew currentVersion >}} has a bug where keys in generated kubeconfig
files are set use RSA despite the feature gate being enabled.
```
(or some similar text)


and then once that merges into _dev-1.31_, propose a new PR to use the text here:
```markdown
switch between the RSA and ECDSA algorithms on the fly or during upgrades. Kubernetes versions before
v1.31 had a bug where keys in generated kubeconfig files were set use RSA, even when you had
enabled the `PublicKeysECDSA` feature gate.
```

Bit more complicated to do, but it's more complete and helps people who are using a release other than the very latest.

---
By “request backports” I mean file an issue asking for docs contributors to backport the docs change to older release docs.

Reply:
Sound good to me.

Reply:
> You can merge this into main (and also request backports)

updated to include this text